plugins/bazaar/src/components/ProjectCard/ProjectCard.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { ItemCardHeader } from '@backstage/core-components';
+import {
+  Card,
+  CardActionArea,
+  CardContent,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import { StatusTag } from '../StatusTag/StatusTag';
+import { Link as RouterLink } from 'react-router-dom';
+import moment from 'moment';
```

Comment:
we are adamantly working to get rid of `moment` actually, see https://github.com/backstage/backstage/blob/master/docs/architecture-decisions/adr010-luxon-date-library.md - you'll have to use luxon instead, see the links in that article and examples elsewhere in the codebase